### DISCUSSION TITLE
Advanced multi-column view: control over column width?

### DISCUSSION THREAD

**Discussion by DanOlner** [mastodon/mastodon#19864] [General]
Advanced multi-column view is fantastic, but the columns are a little narrow. Could there be a way to set width for each user? Edit: p.s. if there's a good newbie contributer guide for Mastodon, I could potentially have a go at this myself? (Though it sounds like a rather complex one to start with - CSS for width would be doable, but adding user option to set it, less so!)

**Comment by sparr** [comment]
https://github.com/mastodon/mastodon/blob/bd220c32f162230d31e99bdabd30aea787a89cfc/app/javascript/styles/mastodon/components.scss#L2283 is where you'd start for the CSS, but you'd also need to handle the other elements that get sized for columns, like https://github.com/mastodon/mastodon/blob/bd220c32f162230d31e99bdabd30aea787a89cfc/app/javascript/styles/mastodon/components.scss#L4207
You could make a userstyle for your browser that changes these, if you want to fix it for yourself immediately.

**Comment by sparr** [comment]
https://github.com/mastodon/mastodon/search?l=Ruby&q=setting_use_blurhash should lead you to the places you'd need to touch to add a new appearance preference. (you'd also need to add localization for the description, but I excluded those from the search)
